src/main/webapp/user_profile.html

```diff
@@ -47,6 +47,13 @@ <h2>User Checklist:</h2>
           <button class="complete" type="button" onclick="goToTask(4)">Task 4</button>
         </li>
       </ul>
+      <form method="GET" action="/user">
+        <button type ="submit" values="users" id="button" onClick="loadAllUsers() return false;">Get Users</button>
```

Comment:
What does it mean that "attribute names must follow format? I looked up some lint rules and it just said lower case with dashes if multi-word names. I'm not sure what has to change here ...

Reply:
recommended html format I believe is `attribute="value"` with no spaces before/after the equals sign.